Makefile

```diff
@@ -24,7 +24,7 @@ WebTag ?= v0.4.0
 DBImg ?= drud/mysql-docker-local-57
 DBTag ?= v0.3.0
 RouterImage ?= drud/nginx-proxy
-RouterTag ?= v0.3.0
+RouterTag ?= router-expose
```

Comment:
This has to be updated before pull. I'd have been more comfortable with the process if this were either set to the expected tag (v0.4.0?) and a provisional v0.4.0 pushed, or else left alone and the instructions telling us to build ddev with `make RouterTag=router-expose`. But every single way is a bit of a cheat to get around all the moving components, so yours is just as good I'd say. But has to be updated.